---
perception_manager/TabletopObject[] objects
